campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -385,6 +624,19 @@
           }
         {% endif %}
 
+        {% if background_image_animate and background_image %}
```

Comment:
Yeah right now animation and special images should probably not be mixed anyway, from a design pov it's prob not good.